patterns/search.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Title: Search
+ * Slug: twentytwentyfive/search
+ * Categories: search
+ * Description: Translatable search form
```

Comment:
This one needs to be hidden. We need to rename the file to `hidden-search` and change the docblocks. For hidden patterns we can avoid the description.

```suggestion
 * Title: Search
 * Slug: twentytwentyfive/hidden-search
 * Inserter: no
```

Reply:
@juanfra this change has been made. I didn't know about the hidden pattern syntax. Thanks for explaining :)